CMakeLists.txt

```diff
@@ -196,9 +196,9 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/fdbclient/BuildFlags.h.in ${CMAKE_CUR
 if (CMAKE_EXPORT_COMPILE_COMMANDS AND WITH_PYTHON)
   add_custom_command(
 	  OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/compile_commands.json
-	  COMMAND $<TARGET_FILE:Python::Interpreter> ${CMAKE_CURRENT_SOURCE_DIR}/build/gen_compile_db.py
+	  COMMAND $<TARGET_FILE:Python::Interpreter> contrib/gen_compile_db.py
```

Comment:
This should have `${CMAKE_CURRENT_SOURCE_DIR}/` in front to make it an absolute path